packages/flutter/lib/src/material/tooltip_theme.dart

```diff
@@ -250,3 +278,19 @@ class TooltipTheme extends InheritedTheme {
   @override
   bool updateShouldNotify(TooltipTheme oldWidget) => data != oldWidget.data;
 }
+
+/// The method of interaction that will trigger a tooltip.
+/// Used in [Tooltip.triggerMode] and [TooltipThemeData.triggerMode]
+enum TooltipTriggerMode {
+  /// Tooltip will not be shown automatically, but can be done manually
+  /// by calling `ensureTooltipVisible`
```

Comment:
Use of automatically can be confusing.
```suggestion
  /// Tooltip will only be shown by calling `ensureTooltipVisible`
```